src/cockpit/389-console/src/lib/ldap_editor/wizards/operations/addCosDefinition.jsx

```diff
@@ -0,0 +1,1053 @@
+import React from 'react';
+import {
+    Alert,
+    Bullseye,
```

Comment:
I know this seems silly, but could you put these imports in alphabetical order?  Trust me if you ever need to go back and make changes (especially big changes) to this file it will make it much easier to find missing/required imports for new components.